.github/workflows/remove-stale-branches.yaml

```diff
@@ -0,0 +1,13 @@
+on:
+  schedule:
+    - cron: "0 0 * * *" # Runs every night at midnight
+  workflow_dispatch: # Allows manual trigger
+
+jobs:
+  remove-stale-branches:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: fpicalausa/remove-stale-branches@v2.0.1
+        with:
+          dry-run: true
```

Comment:
this needs to be false to automatically remove stale branches right?

Reply:
Right, I want to start with dry run.